sponsors/index.html

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: Sponsors
+---
+
+<section class="text-content">
+  <h1>Sponsorships</h1>
+
```

Comment:
All sponsors are more than welcome to give away swag (stickers t-shirts, mugs, socks, etc.).